### DISCUSSION TITLE
Color regions on CIE Chromacity Diagram

### DISCUSSION THREAD

**Discussion by tospe** [colour-science/colour#911] [Q&A]
Hi there, 

I found this categorization of colors on the chromaticity diagram (http://hyperphysics.phy-astr.gsu.edu/hbase/vision/cie.html#c4) 

![imagem](https://user-images.githubusercontent.com/49151314/147090585-a9d38bb0-19d7-4b5a-9f7b-d490c5819868.png)

There is a way of getting this categorization (or similar)?

Thanks in advance!

**Comment by thomasmansencal** [comment]
Hi @tospe,

We unfortunately do not have such a categorisation! The best references I found for that are:

![](https://www.researchgate.net/profile/Kyungah-Choi/publication/280903173/figure/fig4/AS:284463964672000@1444832902075/Left-Approximate-color-regions-on-CIE-1931-x-y-chromaticity-diagram-Gage-et-al.png)

- Left: Gage et al. (1977)
- Right: KS A 0012 (2013)

**Reply by thomasmansencal** [reply]
KS A 0012 has a more precise diagram but no actual values that could be use to define the boundaries analytically:

![image](https://user-images.githubusercontent.com/99779/147204670-789d8f2c-ca97-482a-b3ba-1ac7c5c28cf8.png)

**Reply by tospe** [reply]
Thank you very much for the response.

After you answer i found this [paper ](https://nvlpubs.nist.gov/nistpubs/jres/31/jresv31n5p271_A1b.pdf) which has some analytical values.
I will experiment with these values, and give feedback afterwards.

**Reply by tospe** [reply]
Using the [Color Designations For Lights](https://nvlpubs.nist.gov/nistpubs/jres/31/jresv31n5p271_A1b.pdf) i manage to find another paper (Final report of the O.S.A. subcommittee on the spacing of the Munsell colors) which has some empirical data. This data is also present on the [munsellinterpol](https://github.com/cran/munsellinterpol/tree/master/inst/extdata) R package.  

With this data i manage to obtain diagram which resemble this designations, however i am still improving it.

![points with hue 10 value 5, and chroma > 4 ](https://user-images.githubusercontent.com/49151314/149791044-3bd295f0-70d1-4b08-844c-407d90d69bf3.png)

![interpolation of points with hue 10 and value 5 ](https://user-images.githubusercontent.com/49151314/149790443-d81132e0-e7fb-4b3e-8aac-e8a653f9d8dd.png)

I will keep updating this

**Reply by thomasmansencal** [reply]
Thanks and great find with Kelly (1943)!
